Change NoOps to SQLite across all project folders (#959)
fermyon/developer: content/spin/see-what-people-have-built-with-spin.md

- [Accessing external APIs](https://www.fermyon.com/blog/spin-rest-apis)

## SQLite Storage Using Javascript

> Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a SQLite Storage example (using Javascript).

Reviewer: ```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a SQLite Database example (using JavaScript).
```

> Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a SQLite Storage example (using Javascript).

Author: ```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite Database example (using JavaScript).
```

Author: This is outdated it seems.

> Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides a SQLite Storage example (using Javascript).

Author: ```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite Database example (using JavaScript).
```

> Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite Storage example (using Javascript).

Author: ```suggestion
Check out our [NoOps and Serverless Are the Perfect Pair](https://www.fermyon.com/blog/noops-and-serverless-are-the-perfect-pair) blog article that provides an SQLite database example (using Javascript).
```
